pymedphys/_dicom/rtplan/core.py

```diff
@@ -203,6 +203,28 @@ def get_gantry_angles_from_dicom(dicom_dataset):
     return result
 
 
+def get_leaf_jaw_position_of_type(
```

Comment:
should at least be named get_leaf_jaw_positions_of_type (positions the plural of position)
and maybe get_leaf_jaw_positions_for_type (but that might be a distinction between the Queen's English and American English)

Reply:
Yup, I agree. I'm a little unsure on the of vs for, but I think you're right...